packages/blockchain-api/src/events/EscrowReceived.ts

```diff
@@ -7,8 +7,10 @@ import { Contracts } from '../utils'
 export class EscrowReceived extends TransactionType {
   matches(transaction: Transaction): boolean {
     return (
-      transaction.transfers.length === 1 &&
-      transaction.transfers.containsTransferFrom(Contracts.Escrow)
+      (transaction.transfers.length === 1 &&
```

Comment:
When there are too many conditions, it could be nice to make those checks more readable, so the next dev could understand what is going on:
```
return (
    isEscrowReceivedToEOA(transaction) || 
    isEscrowReceivedToMTW(transaction)
)
```